data/address-geocoders-locators/index.html

```diff
@@ -202,170 +183,3 @@ <h5>Related Resources</h5>
     </p>
   </div>
 </div>
-<div class="grid package">
-  <div class="grid__col grid__col--12-of-12">
-    <h3 id="RoadsAddressLocatorPackage">Roads Address Locator</h3>
-  </div>
-  <div class="grid__col grid__col--12-of-12 package-content">
-    <p style="display: inline-block">
-      <img
-        style="float: left; margin-right: 1em;"
-        src="{% link images/address-geocode.png %}"
```

Comment:
**question**: do we need to check if this image is used anywhere else and can be deleted?

Reply:
good call.  i did a search and did not find the image being used anywhere else so i removed it from the images folder.

Reply:
yep since the build passed you are right. unused.